src/react/forms.jsx

```diff
@@ -5,9 +5,28 @@
 
 'use strict';
 
+var formControlClass = 'mui-form-control',
+    formGroupClass = 'mui-form-group',
+    floatingLabelBaseClass = 'mui-form-floating-label',
+    floatingLabelActiveClass = floatingLabelBaseClass + '-active',
+    animationName = 'mui-form-floating-label-inserted',
+    _supportsPointerEvents;
 
-var jqLite = require('../js/lib/jqLite.js'),
-    forms = require('../js/forms.js');
+var React = require('react');
```

Comment:
I think so, I just did it this way because I'm used to it and pretty new to javascript/React. But maybe there is a way to make it available like this than somebody can include the source code they need and compile it into one script.
There are of course 2 use cases, the user includes react by cdn or whatever, or he compiles everything to one script. It would be great to support both, without React being global.